### ISSUE TITLE
stroom-auth-service fails to connect to a MySQL 8.0 database

### ISSUE THREAD

**Issue by burnalting** [gchq/stroom-auth#172]
When deployed with a MySQL 8.0 database instance, the stroom-auth-service is unable to establish a connection to the database.

Messages are

```
stroom-auth-service    | INFO   [2020-02-23T12:18:25.370Z] [main] org.eclipse.jetty.util.log -  Logging initialized @3542ms to org.eclipse.jetty.util.log.Slf4jLog 
stroom-auth-service    | INFO   [2020-02-23T12:18:25.462Z] [main] io.dropwizard.server.DefaultServerFactory -  Registering jersey handler with root path prefix: / 
stroom-auth-service    | INFO   [2020-02-23T12:18:25.463Z] [main] io.dropwizard.server.DefaultServerFactory -  Registering admin handler with root path prefix: /authenticationServiceAdmin 
stroom-auth-service    | INFO   [2020-02-23T12:18:25.819Z] [main] stroom.auth.util.db.DbUtil -  Ensuring database connection to jdbc:mysql://v6db:3306/auth?useUnicode=yes&characterEncoding=UTF-8 with username authuser 
stroom-auth-service    | Sun Feb 23 12:18:26 GMT 2020 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.
stroom-auth-service    | WARN   [2020-02-23T12:18:26.419Z] [main] stroom.auth.util.db.DbUtil -  Unable to establish database connection due to error: [null] and vendorCode [0], will try again in 100ms, enable debug to see stack trace 
```
Note that the SSL connection notice also appears when deploying using a 5.7 MySQL instance

**Comment by at055612** [comment]
see #173 & https://github.com/gchq/stroom-resources/pull/86

**Comment by at055612** [comment]
Fixed in auth v6.0.28 & stack v6.0.34-2
